include/deal.II/particles/particle.h

```diff
@@ -25,65 +25,65 @@
 DEAL_II_NAMESPACE_OPEN
 
 /**
- * A namespace that contains all classes that are related to the particle
- * implementation, in particular the fundamental Particle class.
+ * A namespace for all type definitions related to particles.
  */
-namespace Particles
+namespace types
 {
   /**
-   * A namespace for all type definitions related to particles.
+   * Typedef of cell level/index pair. TODO: replace this by the
+   * active_cell_index.
    */
-  namespace types
-  {
-    /**
-     * Typedef of cell level/index pair. TODO: replace this by the
-     * active_cell_index.
-     */
-    typedef std::pair<int, int> LevelInd;
+  typedef std::pair<int, int> LevelInd;
```

Comment:
Is this typedef ever used in user-facing code, or is it only internal?

Reply:
I'm asking because if it is, maybe we should give the typedef a better name. And if it isn't, maybe we should just move the typedef to an internal namespace...

Reply:
Yes we should make this internal. Let me think about the best way to do it. The only place it is currently used is in `insert_particles`, and the only reason to use it is to keep interfaces in ASPECT intact. I will try to come up with a better way.